src/element/textWysiwyg.tsx

```diff
@@ -566,6 +567,12 @@ export const textWysiwyg = ({
 
   // handle updates of textElement properties of editing element
   const unbindUpdate = Scene.getScene(element)!.addCallback(() => {
+    if (isTextElement(element)) {
+      const updatedElement = Scene.getScene(element)!.getElement(
+        element.id,
+      ) as ExcalidrawTextElement;
+      prevText = updatedElement.text;
```

Comment:
doesn't this assignment make the `prevText` variable equivalent to `updatedElement.text`?